nototools/shape_diff.py

```diff
@@ -88,27 +90,37 @@ def find_rendered_diffs(self, font_size=24, stats=None):
         cmp_png = 'tmp_cmp.png'
         diffs_filename = 'tmp_diffs.txt'
 
-        with open(diffs_filename, 'w') as ofile:
-            for name in ordered_names:
-
-                # ignore null character, and characters without unicode values
-                unival = self.reverse_cmap.get(name, 0)
-                if unival == 0:
-                    continue
-
-                strin = unichr(unival)
-                subprocess.call([
-                    'hb-view', '--font-size=%d' % font_size,
-                    '--output-file=%s' % a_png, path_a, strin])
-                subprocess.call([
-                    'hb-view', '--font-size=%d' % font_size,
-                    '--output-file=%s' % b_png, path_b, strin])
+        for name in ordered_names:
+            strin, strin_b = [
+                hb_input.hb_input_from_name(f, name, self.reverse_cmap)
```

Comment:
When building the reverse cmaps it will report disparity. But right now it's just taking one font's cmap where it should be taking the intersection.